app.js

```diff
@@ -1,18 +1,27 @@
 const express = require('express');
-const { getAllCategories } = require('./controllers/controller-games');
+const { getAllCategories, getReviewByID } = require('./controllers/controller-games');
 
 const app = express();
 
 app.use(express.json());
 
 app.get('/api/categories', getAllCategories);
 
+app.get('/api/reviews/:review_id', getReviewByID);
+
+// app.all('/api/reviews/*', (req, res) => {
```

Comment:
I'd try to avoid commented out code in your PRs, if it's not relevant to the current ticket then you don't need to put in in yet :)